scripts/load_all_fund_rounds.py

```diff
@@ -0,0 +1,18 @@
+from scripts.fund_round_loaders.load_cof_r2 import main as load_cof_r2
+from scripts.fund_round_loaders.load_cof_r3w1 import main as load_cof_r3w1
+from scripts.fund_round_loaders.load_cof_r3w2 import main as load_cof_r3w2
+from scripts.fund_round_loaders.load_ns_r2 import main as load_ns_r2
+
+
+def main() -> None:
+    load_cof_r2()
+    load_cof_r3w1()
+    load_cof_r3w2()
+    load_ns_r2()
```

Comment:
Could we do some kind of inspection thing to find all the `load_xxx` methods and just run all of them? Wouldn't need to remember to update this when we add a new fund/round then.

Reply:
another good shout, let me investigate this 👍